stack.yaml

```diff
@@ -91,6 +94,7 @@ extra-deps:
 
   # Windows only
   - Win32-2.6.2.0
+  - hedgehog-1.0.1@sha256:b65b1ebde7b7fe8e693070916e45aa2ed63eea5e3f84ac3a8f9a0b7f8fcf0337
```

Comment:
Why is this needed? The `snapshot.yaml` in `cardano-prelude` mentions `hedgehog-1.0.2`, so this is overriding it. Also, the `cabal.project` does not mention this constraint.

Reply:
My bad. I probably first tried stack on a pretty old commit which mentioned `hedgehog-1.0`

Reply:
Related: #1796